compiler/z/codegen/OMRCodeGenerator.cpp

```diff
@@ -457,6 +457,15 @@ TR_S390ProcessorInfo::checkZ13()
    }
 
 ////////////////////////////////////////////////////////////////////////////////
+// TR_S90ProcessorInfo::checkZ14: returns true if it's z14, otherwise false.
```

Comment:
Fixed in cb358ed. I also fixed a few other occurrences while in this area of code.